test/files/neg/t2102.check

```diff
@@ -1,6 +1,6 @@
 t2102.scala:2: error: type mismatch;
- found   : java.util.Iterator[Int]
- required: scala.collection.Iterator[_]
+ found   : Iterator[Int] (in java.util)
+ required: Iterator[_] (in scala.collection)
```

Comment:
Love the first idea at least for sure. I think it definitely deserves including.